sympy/stats/tests/test_rv.py

```diff
@@ -117,7 +117,7 @@ def test_H():
     assert H(D, D > 2) == log(2)
     assert comp(H(G).evalf().round(2), 1.39)
 
-
+@XFAIL # sample function now returns array (currently for crv)
```

Comment:
remove this line immediately, I don't want to see any `@XFAIL` added to the tests. If you need to add `@XFAIL`, it means your PR will cause damage to SymPy.